csunplugged/tests/topics/loaders/test_lessons_loader.py

```diff
@@ -0,0 +1,457 @@
+                "<ProgrammingExercise: Exercise 2>",
+            ],
+            ordered=False,
+        )
+
+    def test_learning_outcomes(self):
+        config_file = os.path.join(self.loader_name, "learning-outcomes.yaml")
+        complete_config_file_path = os.path.join(self.test_data.LOADER_ASSET_PATH, config_file)
+        lessons_structure = self.test_data.load_yaml_file(complete_config_file_path)
+
+        topic = self.test_data.create_topic(1)
+        unit_plan = self.test_data.create_unit_plan(topic, 1)
+        self.test_data.create_learning_outcome(1)
+        self.test_data.create_learning_outcome(2)
+
+        lesson_loader = LessonsLoader(
+            config_file,
+            self.load_log,
+            lessons_structure,
+            topic,
+            unit_plan,
+            self.test_data.LOADER_ASSET_PATH
+        )
+        lesson_loader.load()
+        self.assertQuerysetEqual(
+            Lesson.objects.get(slug="lesson-1").learning_outcomes.all(),
+            [
+                "<LearningOutcome: Outcome 1>",
+                "<LearningOutcome: Outcome 2>",
+            ],
+            ordered=False,
+        )
+
+    def test_classroom_resources(self):
+        config_file = os.path.join(self.loader_name, "classroom-resources.yaml")
+        complete_config_file_path = os.path.join(self.test_data.LOADER_ASSET_PATH, config_file)
+        lessons_structure = self.test_data.load_yaml_file(complete_config_file_path)
+
+        topic = self.test_data.create_topic(1)
+        unit_plan = self.test_data.create_unit_plan(topic, 1)
+
+        lesson_loader = LessonsLoader(
+            config_file,
+            self.load_log,
+            lessons_structure,
+            topic,
+            unit_plan,
+            self.test_data.LOADER_ASSET_PATH
+        )
+        lesson_loader.load()
+        self.assertEqual(
+            Lesson.objects.get(slug="lesson-1").classroom_resources,
+            ["Pens", "Paper"],
+        )
+
+    def test_multiple_lessons(self):
+        config_file = os.path.join(self.loader_name, "multiple-lessons.yaml")
+        complete_config_file_path = os.path.join(self.test_data.LOADER_ASSET_PATH, config_file)
+        lessons_structure = self.test_data.load_yaml_file(complete_config_file_path)
+
+        topic = self.test_data.create_topic("1")
+        unit_plan = self.test_data.create_unit_plan(topic, "1")
+
+        lesson_loader = LessonsLoader(
+            config_file,
+            self.load_log,
+            lessons_structure,
+            topic,
+            unit_plan,
+            self.test_data.LOADER_ASSET_PATH
+        )
+        lesson_loader.load()
+        print(Lesson.objects.all())
+        self.assertQuerysetEqual(
+            Lesson.objects.all(),
+            [
+                "<Lesson: Lesson 1>",
+                "<Lesson: Lesson 2>",
+                "<Lesson: Lesson 3>",
+            ],
+            ordered=False,
+        )
```

Comment:
What about the `generated-resources:` field. (I'm guessing this is purposely missed.)